source/constructs/api/catalog/service.py

```diff
@@ -1236,8 +1236,12 @@ def fill_catalog_labels(catalogs):
 
 def rebuild_catalog_labels(catalogs):
     result = []
+    if len(catalogs) == 0:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`

[Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)

Similar issue at line number 1248.